### PR TITLE
DOCSP-42284: v1.5 release

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-42284
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/pymongo-arrow/DOCSP-42284-1.5-release/whats-new/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -20,7 +20,10 @@ What's New
 Changes in Version 1.5.0
 ------------------------
 
-- Added support for PyArrow 17.0.
+- Adds support for PyArrow 17.0
+- Fixes a bug where adding a struct with an ``ObjectId`` value to a list
+  raised an error
+- Adds support for nested ``ObjectID`` values in Polars conversions
```
Comment (jordan-smith721) [inline source/whats-new.txt]: I think "D" should be lowercase here?

```suggestion
- Adds support for nested ``ObjectId`` values in Polars conversions
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ one small thing

File: source/whats-new.txt [hunk 2]
```diff
@@ -20,7 +20,10 @@ What's New
 Changes in Version 1.5.0
 ------------------------
 
-- Added support for PyArrow 17.0.
+- Adds support for PyArrow 17.0
+- Fixes a bug where adding a struct with an ``ObjectId`` value to a list
```
Comment (caseyclements) [inline source/whats-new.txt]: These are the same. Support for nested ObjectId and the issue about it.

**Review by caseyclements — CHANGES_REQUESTED** [review]
I'd remove line 24 on whats-new.

File: source/whats-new.txt (hunk 2 above)
Comment (norareidy) [reply source/whats-new.txt]: Removed!

**Review by caseyclements — APPROVED** [review]
LGTM

**Pull request merged**